[IMP] Accounting: follow-up reports (#4423)
odoo/documentation: content/applications/finance/accounting/receivables/customer_payments/followup.rst

=============

To configure a :guilabel:`Follow-Up Action`, go to :menuselection:`Accounting --> Configuration -->
Follow-up Levels`. Several follow-up actions are available by default, but you can

> :guilabel:`Create` new ones if required. You can change its **name** and the **number of days**
> after which the follow-up actions are triggered. The follow-up :guilabel:`Actions` available are:

Reviewer: - no need for that guilabel since you're not referring to an UI element on the screen. You're just saying that it can be created.
- "new ones" = plural, so the next sentence is wrong.
```suggestion
create new ones if required, and change their names and the number of days
after which the follow-up actions are triggered. The follow-up :guilabel:`Actions` available are:
```

- :guilabel:`Send an email`;
- :ref:`Send an SMS message <pricing/pricing_and_faq>`;
- :guilabel:`Print a letter`;
- :ref:`Send a letter <customer_invoices/snailmail>`;
- :guilabel:`Manual action` (creates a task).

It is possible to automatically send a reminder by enabling the :guilabel:`Auto Execute` option, and
attaching the *open* invoice(s) by enabling :guilabel:`Join open invoices`, within a specific

> :guilabel:`Follow-Up Action`.

Reviewer: ```suggestion
follow-up action.
```

Depending on the :guilabel:`Actions` enabled, the :guilabel:`Message` tab contains different message
options. The :guilabel:`Email Subject` and the :guilabel:`Description` or content of the email
automatically used. If you selected :guilabel:`SMS`, the content of the SMS text. In case of
:guilabel:`Manual Action`, you can :guilabel:`Assign a Responsible` person to manage that follow-up
and the :guilabel:`Manual Action Type` (:guilabel:`Email`, :guilabel:`Call`, etc.), as well as input

> a description of the :guilabel:`Action To Do`.

Reviewer: This sentence "sounds" a bit weird, especially with the mix of guilabel and words.

```If you enabled :guilabel:`Manual Action`, you can assign someone in the guilabel:`Assign a Responsible` field to manage that follow-up ....```

.. note:: You should **not** delete text between `%(text)`, as it automatically fetches the
   partner and company's name.
.. tip::
   To send a reminder before the actual due date is reached, set a negative number of due days.

Follow-up reports
=================

Overdue invoices you need to follow up on are available in :menuselection:`Accounting --> Customers
--> Follow-up Reports`. By default, Odoo filters by :guilabel:`Customer Invoices` that are
:guilabel:`In need of action`.

> When selecting an invoice, you see all of the customer's unpaid invoices (overdue or not), as well
> as their payments. The due dates of late invoices appear in red. Select the invoices that are *not*

Reviewer: ```suggestion
When selecting an invoice, you see all of the customer's unpaid invoices (overdue or not), and
their payments. The due dates of late invoices appear in red. Select the invoices that are *not*
```

> late by clicking :guilabel:`Excluded` column to exclude them from the reminder you send.

Reviewer: **the** column
